src/siteconfig/tests/test_models.py

```diff
@@ -103,3 +107,15 @@ def test_SiteConfig_cache_expires(self):
         cache_time_expiration = localtime() + timedelta(days=1)
         with freeze_time(cache_time_expiration, tz_offset=0):
             self.assertIsNone(cache.get(SiteConfig.cache_key()))
+
+    def test_get_deck_owner(self):
+        """
+            Test to make sure the default deck owner is the correct one
+        """
```

Comment:
Be more descriptive in the docstring about what specifically is being tested.  I think should be "Test to make sure new decks have the expected default deck_owner, as set in settings.py via .env`